### PR TITLE
add precourse challenges

### PR DESCRIPTION
**Pull request by luislourencos**

### CODE REVIEW

File: staff/sergio-luis/precourse/airlines/airlines/index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+}
+//Show fligths who make scale
+function scale() {
+    var i = 0;
+    console.log('VUELOS CON ESCALA :\n');
+    for (i; i < flights.length; i++) {
+        if (flights[i].scale) {
+            console.log('Vuelo nº: ' + flights[i].id + ' - El vuelo con origen: ' + flights[i].from + ' y con destino: ' + flights[i].to + ' tiene un coste de ' + flights[i].cost + '€, y efectua escala\n');
+        };
+    }
```
Comment (Pauatro) [inline staff/sergio-luis/precourse/airlines/airlines/index.js:95]: you semicolons are not absolutely everywhere, I'm very very sad :(

Comment (luislourencos) [reply staff/sergio-luis/precourse/airlines/airlines/index.js:95]: No problem y will fixed now!

File: staff/sergio-luis/precourse/bingo/index.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        arrNumberBingoCart.push(number);
+        bingoCard[i].number = number;
+        bingoCard[i].matched = false;
+    }
+    showCard()
+};
+
+// Function showCart() by console
+function showCard() {
+    console.log('***********************************')
```
Comment (Tanane) [inline staff/sergio-luis/precourse/bingo/index.js:66]: Use a for loop or a foreach to loop through each position instead of typing manually!

**Comment by Tanane** [comment]
I liked the connect 4! You've put so much work on it

File: staff/sergio-luis/precourse/bingo/index.js (hunk 2 above)
Comment (luislourencos) [reply staff/sergio-luis/precourse/bingo/index.js]: Ok changed

Comment (luislourencos) [reply staff/sergio-luis/precourse/bingo/index.js]: Thanks!

**Pull request merged**